crates/anemo/src/network/mod.rs

```diff
@@ -154,22 +154,28 @@ impl Builder {
             }
             Err(result.unwrap_err())
         })()?;
-        if let Some(send_buffer_size) = quic_config.socket_send_buffer_size {
-            socket.set_send_buffer_size(send_buffer_size)?;
-            let buf_size = socket.send_buffer_size()?;
-            if buf_size < send_buffer_size {
-                // Linux doubles requested size, so allow anything greater.
-                let msg = format!(
+        let socket_send_buf_size =
+            if let Some(send_buffer_size) = quic_config.socket_send_buffer_size {
+                socket.set_send_buffer_size(send_buffer_size)?;
+                let buf_size = socket.send_buffer_size()?;
+                if buf_size < send_buffer_size {
+                    // Linux doubles requested size, so allow anything greater.
+                    let msg = format!(
```

Comment:
I think L164-L165 need to be indented.

Reply:
good catch, thanks! wish cargo fmt worked for that sort of thing

Reply:
FYI it turns out cargo fmt refuses to let this be correct. probably a bug but was easiest to just go with the bad formatting rather than fight it.